books.css

```diff
@@ -0,0 +1,55 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+h1 {
+  margin: 0 auto;
+  text-align: center;
+}
+
+.section1,
+.section2,
+.section3 {
+  display: flex;
+
+  /* justify-content: center; */
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean. Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.